src/jats/jats-exporter.ts

```diff
@@ -1109,7 +1065,7 @@ export class JATSExporter {
 
         return ['sec', attrs, 0]
       },
-      hard_break: () => ['break'],
+      hard_break: () => '',
```

Comment:
Not sure how it worked before. Will that still ensure a hard break to be properly exported? Or is it not really translatable into JATS?

Reply:
Although they are similar, a hard_break is allowed in places where <break> isn't (e.g. paragraphs), so I opted to remove the mapping for the time being since a missing <break> is not going to change the meaning of the output. I will file a ticket to understand more what hard_break does.